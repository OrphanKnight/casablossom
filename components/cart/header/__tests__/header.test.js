import { render } from '@testing-library/react';
import Checkout from '../index';

it('renders Checkout component', () => {
  const { container } = render(<Checkout />);
  expect(container).toMatchSnapshot();
})