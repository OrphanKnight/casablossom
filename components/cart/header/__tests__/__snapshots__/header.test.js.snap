// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Checkout component 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="header__container"
    >
      <div
        class="header__left"
      >
        <a
          href="/"
        >
          <img
            alt=""
            src="../../../images/logo.png"
          />
        </a>
      </div>
      <div
        class="header__right"
      >
        <a
          href="/browse"
        >
          Continue Shopping
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M8 5v14l11-7z"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;
